### DISCUSSION TITLE
How to import sizing and spacing only?

### DISCUSSION THREAD

**Discussion by zshlomyz** [twbs/bootstrap#37150] [Q&A] [answered]
Hi, 
I would like to import only the sizing and spacing.
I can't find how to do that except of importing the entire library.
I've tried to import the utilities part but it didn't work.
Can you help, please?
Thanks

**Comment by mdo** [comment, marked as answer]
You can start with our utilities build, which only compiles our utility classes and some global CSS variables. https://github.com/twbs/bootstrap/blob/main/scss/bootstrap-utilities.scss

From there, use our utilities API to customize the utilities you need. https://getbootstrap.com/docs/5.2/utilities/api/

**Reply by zshlomyz** [reply]
Thanks! BTW - I can't get the margin right classes (mr-<x>) to work. Is that still supported?

**Reply by mdo** [reply]
Margin and padding classes were renamed to use logical property names. Check the migration guide for more info :).

**Reply by zshlomyz** [reply]
Thanks!
